src/Components/Header/Header.js

```diff
@@ -1,6 +1,19 @@
+import React, { Component } from "react";
+import {Link} from 'react-router-dom';
+import '../Header/Header.css'
+
 const Header = () => {
     return (
-        <div>Header</div>
+        <div>
+            <h1>Header</h1>
+
+    <Link to='/home' className="links"> <button> Home </button> </Link>
```

Comment:
home is the landing page so it's route is just '/' if you want to change that when you get a chance